### PR TITLE
Add disable flag for plugin Install.

### PR DESCRIPTION
**Pull request by anusha-ragunathan**

### CODE REVIEW

**Comment by vdemeester** [comment]
LGTM 🐯

**Comment by justincormack** [comment]
LGTM

File: api/client/plugin/install.go [hunk 1]
```diff
@@ -35,6 +36,7 @@ func newInstallCommand(dockerCli *client.DockerCli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.BoolVar(&options.grantPerms, "grant-all-permissions", true, "grant all permissions necessary to run the plugin")
+	flags.BoolVar(&options.disable, "disable", false, "do not enable the plugin on install")
```
Comment (tiborvass) [inline api/client/plugin/install.go]: disable or disabled?

Comment (icecrime) [inline api/client/plugin/install.go]: I prefer `--disabled`.

**Pull request merged**